### ISSUE TITLE
fdi-app.fiduciarydecisions.com: breakage

### ISSUE THREAD

**Issue by DanielFDi** [uBlockOrigin/uAssets#26438]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.). Comment in the old issue threads even when they are closed.
- [X] I did not remove any of the [default filter lists](https://github.com/user-attachments/assets/1879fc88-09bc-4de4-9e2f-3834a6ab396a), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://fdi-app.fiduciarydecisions.com/rollovers`

### Category

breakage

### Description

Home page loads fine after logging in successfully.  Pages load blank after clicking on certain navigation links.
 (Rollovers, Data, Tools menu -> To-Dos, Form ADV, Data Status).

### Other extensions used

IBM Equal Accessibility Checker
Keeper
React Developer Tools

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

<img width="666" alt="Screenshot 2024-12-12 100627" src="https://github.com/user-attachments/assets/bf07ab6c-6896-499e-a733-93de88437757" />
<img width="668" alt="Screenshot 2024-12-12 100734" src="https://github.com/user-attachments/assets/63fb3b28-5907-462a-985f-08815cd5794e" />

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.61.2
Chromium: 131
filterset (summary):
 network: 141204
 cosmetic: 49781
 scriptlet: 23269
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 40926-251, 1h.27m
  ublock-badware: 12400-1, 27m Δ
  ublock-privacy: 1637-22, 1h.27m
  ublock-unbreak: 2541-0, 1h.27m
  ublock-quick-fixes: 207-0, 1h.27m
  easylist: 78484-626, 27m Δ
  easyprivacy: 53300-639, 17m
  urlhaus-1: 22895-0, 1h.27m
  plowe-0: 3545-0, 1h.27m
filterset (user): [empty]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 5810 ms
 maxAssetCacheWait: 70 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 8
 network:
  fiduciarydecisions.com: 5
  google-analytics.com: 1
  googletagmanager.com: 1
  marmelab.com: 1
```
</details>

**Comment by mapx-** [comment]
- open the uBO's logger: https://github.com/gorhill/uBlock/wiki/The-logger
- repeat the steps to get the issue
- go back into the uBO's logger and then copy and paste here all blocked requests

or tell us if you want share a test account

**Comment by DanielFDi** [comment]
https://fdi.fiduciarydecisions.com/a/app/analytics/analytics.html?v=47 
forgot one sorry..

**Comment by mapx-** [comment]
please provide the complete rows in the logger (including the filters on the left side)

**Comment by DanielFDi** [comment]
<img width="952" alt="image" src="https://github.com/user-attachments/assets/8446f1e9-ec46-4ba9-8fae-381aca7c732a" />

**Comment by mapx-** [comment]
test 1 by 1 these filters and if still broken try combinations:

```adb
@@||fdi.fiduciarydecisions.com/a/app/analytics/analytics.html$xhr,1p
@@||fdi.fiduciarydecisions.com/v/app/components/*/Analytics.js$script,1p
@@||fdi.fiduciarydecisions.com/a/lib/gtag/gtag.js$script,1p
```

**Comment by DanielFDi** [comment]
when I apply all 3 together, it fixes all of the blank page issues.

**Comment by mapx-** [comment]
Did you test them as I asked "1 by 1" ?

**Comment by DanielFDi** [comment]
yes.  when I applied the 2nd filter only, it only solved 1 blank page issue, but did not fix the others.  I tried each one by itself and it did not fix all blank pages.  Only when I applied all 3 filters, did all the blank pages content load correctly.

**Comment by DanielFDi** [comment]
is there anything else I can do?  What is the next step?

**Comment by mapx-** [comment]
Eayprivacy list maintainer has been notified

**Comment by DanielFDi** [comment]
thanks for all the help and the quick response time.
